plugins/source/aws/resources/services/ram/resource_share_associated_resources_fetch.go

```diff
@@ -18,10 +18,9 @@ func fetchRamResourceShareAssociatedResources(
 ) error {
 	input := &ram.GetResourceShareAssociationsInput{
 		AssociationType: types.ResourceShareAssociationTypeResource,
-		ResourceArn:     parent.Item.(types.ResourceShare).ResourceShareArn,
-		MaxResults:      aws.Int32(500),
+		// ResourceArn:     parent.Item.(types.ResourceShare).ResourceShareArn,
```

Comment:
why do we skip this? in this case it's no longer a relation

Reply:
It is now a top level table